sdk/search/Azure.Search.Documents/src/Generated/Models/SearchIndex.cs

```diff
@@ -30,11 +30,11 @@ public partial class SearchIndex
         /// <param name="normalizers"> The normalizers for the index. </param>
         /// <param name="encryptionKey"> A description of an encryption key that you create in Azure Key Vault. This key is used to provide an additional level of encryption-at-rest for your data when you want full assurance that no one, not even Microsoft, can decrypt your data in Azure Cognitive Search. Once you have encrypted your data, it will always remain encrypted. Azure Cognitive Search will ignore attempts to set this property to null. You can change this property as needed if you want to rotate your encryption key; Your data will be unaffected. Encryption with customer-managed keys is not available for free search services, and is only available for paid services created on or after January 1, 2019. </param>
         /// <param name="similarity"> The type of similarity algorithm to be used when scoring and ranking the documents matching a search query. The similarity algorithm can only be defined at index creation time and cannot be modified on existing indexes. If null, the ClassicSimilarity algorithm is used. </param>
-        /// <param name="Etag"> The ETag of the index. </param>
-        internal SearchIndex(string name, IList<SearchField> Fields, IList<ScoringProfile> scoringProfiles, string defaultScoringProfile, CorsOptions corsOptions, IList<SearchSuggester> suggesters, IList<LexicalAnalyzer> analyzers, IList<LexicalTokenizer> tokenizers, IList<TokenFilter> tokenFilters, IList<CharFilter> charFilters, IList<LexicalNormalizer> normalizers, SearchResourceEncryptionKey encryptionKey, SimilarityAlgorithm similarity, string Etag)
+        /// <param name="etag"> The ETag of the index. </param>
```

Comment:
Should this be 'eTag' to adhere to camel casing?

Reply:
While I would have coded as eTag manually,there are existing code on this.

https://docs.microsoft.com/en-us/dotnet/api/system.web.httpcachepolicy.setetag?view=netframework-4.8

Reply:
I see both usages in our .NET SDKs, with more using the pascal casing. It would be good to standardize on `eTag`.
> grep -r "\\<eTag\\>" --include="\*.netstandard2.0.cs" | Measure-Object -line

68
> grep -r "\\<etag\\>" --include="\*.netstandard2.0.cs" | Measure-Object -line

17